### PR TITLE
Update Springshare apps with new MIT branding

### PR DESCRIPTION
**Pull request by jazairi**
#### Why these changes are being introduced:

The Institute has refreshed its branding.

#### Relevant ticket(s):

* https://mitlibraries.atlassian.net/browse/PW-71
* https://mitlibraries.atlassian.net/browse/PW-65

#### How this addresses that need:

This updates the `custom-header`, `custom-footer`, and
`custom-head.html` files for all Springshare properties, excluding
LibStaff and the beta A-Z Databases. (The latter is ticketed
separately and will come in a subsequent commit.)

The all three files should be identical for LibGuides, LibAnswers,
and LibCal. The other two properties have slight differences:

* LibAnswers DDC's header has no nav. The header consists of
the MITL logo and the new MIT logo.
* LibWizard has no footer. As such, the header nav is replaced
with links to the libraries' privacy and accessibility policies.

Otherwise, the only difference should be the app name in the
'Local overrides' comment.

#### Side effects of this change:

* Upstream changes have been pulled from the compiled WP styles. 
See commit 817df1a for details.
* The new favicons have been added to the LibGuides `custom-head`.
(These were added to the other `custom-head` files in the
last commit, which pulled in upstream changes from WordPress.)
* The nav html has been removed from LibAnswers DDC, as it does
not have a nav and shouldn't have a hamburger menu in mobile.
* Hover/focus highlights have been fixed to extend to the full
height of the header.
* Despite loading the favicons at the top of `custom-head`,
Springshare prioritizes the Springshare favicon. It's not clear
to me how to fix this.
* To remain consistent with our previous practices, all of the
changes were made in the 'Local overrides' sections of the
`custom-head` files. We should consider a longer term solution
of trimming the `custom-heads` to what is actually needed,
and incorporating the local overrides so that we aren't repeating
media queries and rules further down in the files.

### CODE REVIEW

**Comment by jazairi** [comment]
This has largely passed stakeholder review at this point, but I still need approval on the LibAnswers DDC header. I don't anticipate any substantial changes forthcoming, though.

**Comment by matt-bernhardt** [comment]
Quick question as I try and review this - I know there's the test group on LibGuides, which I think you've been using to develop these changes. I don't, however, remember there being similar groups for other Springshare platforms (LibWizard, LibCal etc) ? Or do these exist and I've forgotten / missed them?

**Comment by jazairi** [comment]
@matt-bernhardt There are not, to my knowledge, similar groups for the other Springshare apps. I've been using the LibGuides test group to preview everything. So far, the only oddity we've uncovered is that the LibAnswers breadcrumbs in the LibGuides test group are styled differently than we'd like. I don't expect that this will be an issue in LibAnswers, as I pasted the custom head/header into devtools in both prod apps (LibAnswers and LibAnswers DDC), and the breadcrumbs looked normal.

**Review by matt-bernhardt — COMMENTED** [review]
This looks pretty good from what I can parse - which is not a comment about anything you're doing, but more that I really wish there was a better way to confirm changes in practice before we deploy them in production.

My big question, before approving, is to confirm what's on the Libguides test group right now - because it doesn't look right, but I'm not sure what state it is meant to be right now. The footer in particular looks un-styled but that may be down to something else than this change.

Ultimately, if things look alright when in place, then I'll be 👍 pretty much regardless.

**Review by matt-bernhardt — APPROVED** [review]
Thanks for the help identifying what the state of the test group was. When I put the libguides materials there now, everything looks good - and the various properties look similar enough to each other that I don't have any concerns about those specifically.

Thanks for your patience while I worked through this.

:shipit:

File: libanswers/custom-header.html [hunk 1]
```diff
@@ -146,10 +145,10 @@ <h3 class="heading-col">About the Libraries</h3>
           </ul>
         </div>
         <div class="col-2 flex-item">
-          <h3 class="heading-col">News, events, & exhibits</h3>
-          <a href="https://libraries.mit.edu/events">Classes & events</a>
+          <h3 class="heading-col">News, events, &amp; exhibits</h3>
```
[the remark this answers is not in the document]
Comment (jazairi) [reply libanswers/custom-header.html:148]: I highly doubt it does make a difference, and in this case I suspect the culprit was an HTML prettifier. If someone were to prettify it in the opposite direction, I wouldn't object. :)

**Comment by jazairi** [comment]
Thanks for sticking with this one, Matt!

**Comment by jazairi** [comment]
These changes are now live in prod.

**Pull request merged**